.github/scripts/cmd/cmd.py

```diff
@@ -262,9 +262,10 @@ def main():
         command = f"cargo +nightly fmt"
         print(f'Formatting with `{command}`')
         nightly_status = os.system(f'{command}')
+        os.system("cargo metadata --format-version 1 > /dev/null") # update the lockfile
+        zepter_status = os.system('zepter run --config .config/zepter.yaml')
```

Comment:
@ggwpez do we need this `cargo metadata`?

Reply:
It can otherwise look like zepter hangs, since it does a cargo metadata call internally.  
So in CI it is fine to do this, just to see the progress when cargo metadata is slow.